_module_templates/standard_module_template.md

```diff
@@ -3,76 +3,76 @@
 author:   Your Name
 email:    email@chop.edu
 version:  0.0.0
+module_type: standard
 module_template_version: 3.0.0
 language: en
 narrator: UK English Female
+
 title: Module Title
+
 comment:  This is a short, focused description of the module.
+
 long_description: This is a longer description, which should be understandable for a lay audience. It will print under "Is this module right for me?" in the overview.
+
 estimated_time: This is rough guess of how long it might take a learner to work through the module. It will print under "Estimated time to completion" in the overview
```

Comment:
```suggestion
estimated_time_in_minutes: This is rough guess of how long it might take a learner to work through the module. It will print under "Estimated time to completion" in the overview. Valid values are any integer 1-60. 
```

in concordance with suggestion described on module_macros.md